test/controllers/PaymentHistoryControllerSpec.scala

```diff
@@ -314,6 +314,16 @@ class PaymentHistoryControllerSpec extends ControllerBaseSpec {
         document.select("h1").first().text() shouldBe "Sorry, there is a problem with the service"
       }
     }
+
+    "the user is insolvent and not continuing to trade" should {
+
+      "return 403 (Forbidden)" in new Test {
+        override val authCall = true
+        override lazy val authResult: Future[~[Enrolments, Option[AffinityGroup]]] = successfulAuthResult
```

Comment:
No need to override `authResult` as it's already set to something identical to `successfulAuthResult` by default in the trait